doc/README.md

```diff
@@ -63,14 +72,14 @@ The Bitcoin repo's [root README](/README.md) contains relevant information on th
 - [BIPS](bips.md)
 - [Dnsseed Policy](dnsseed-policy.md)
 - [Benchmarking](benchmarking.md)
-- [Internal Design Docs](design/)
```

Comment:
This exists

Reply:
Added this documentation reference back in to the README.md file.